Description: Build WITH_WRAP on Debian Forky fails
 This is not fixed yet!
Author: Joachim Zobel <jz-2017@heute-morgen.de>
Bug: https://github.com/eclipse-mosquitto/mosquitto/issues/3547
Last-Update: 2026-03-21
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -126,7 +126,7 @@
 option(USE_LIBWRAP "Include tcp-wrappers support?" OFF)
 
 if(USE_LIBWRAP)
-	target_sources(mosquitto wrap)
+	target_sources(mosquitto PRIVATE wrap)
 	target_compile_definitions(mosquitto PRIVATE "WITH_WRAP")
 endif()

 
